priam/src/main/java/com/netflix/priam/aws/SDBInstanceData.java

```diff
@@ -275,7 +276,7 @@ private PriamInstance transform(Item item) {
                     ins.setHost(att.getValue());
                     break;
                 case Attributes.LOCATION:
-                    ins.setRegionName(att.getValue());
+                    ins.setLocation(Location.from(att.getValue()));
```

Comment:
I'll be the first to admit this is not ideal.  One could easily argue that the correct approach would be to create a new column in the SimpleDB table exclusively for the data center suffix.  However, I didn't go with the approach for the following reasons:

1. The existing SimpleDB table is used across many teams using Cassandra, not just Emo, so any changes to the DDL could potentially have widespread consequences.
2. The existing column name is "location".  Prior to the change it held the region name.  Since the column is named "location" and not "region" it seems logical that the intent was for this column to be an extensible identifier for the location and not simply the region.
3. The approach taken here is fully backwards compatible with the existing Priam data and application; no intermediate bi-compatible release is necessary.